src/SprykerEco/Zed/Heidelpay/Business/Mapper/QuoteToHeidelpayRequest.php

```diff
@@ -82,9 +84,43 @@ protected function mapQuoteInformation(QuoteTransfer $quoteTransfer, HeidelpayRe
                 ->setIdOrder($this->generateQuoteId($quoteTransfer))
         );
 
+        $customerRegistrationDate = $this->findCustomerRegistrationDate($quoteTransfer->getCustomer());
+
+        $heidelpayRequestTransfer->setRiskInformation(
+            (new HeidelpayRiskInformationTransfer())
```

Comment:
Mb you can move this to the separate method?

Reply:
Moved to `mapCustoemrInformation()`.